lib/pdf_text.dart

```diff
@@ -120,9 +121,12 @@ class PDFDoc {
     for (var i = 0; i < missingPagesNumbers.length; i++) {
       pageAt(missingPagesNumbers[i])._text = missingPagesTexts[i];
     }
-    String text = "";
-    _pages.forEach((page) => text += "${page._text}\n");
-    return text;
+    /// Removed the \n added at the end of each page here (potentially a breaking change!).
+    /// Since every page can be retrieved individually, the client knows exactly
+    /// where it begins and where it ends, therefore there is no benefit of
+    /// introducing an artificial page separator that is not part of the pdf
+    /// document per se
+    return _pages.fold<String>("", (pv, page) => "$pv${page._text}");
```

Comment:
Yes, removing the "\n" makes sense. Starting the fold with "" means preceding the entire text with a "", right?

Reply:
That means that the fold will start concatenating to an empty string, so essentially yes the entire text will start with an empty string.